### DISCUSSION TITLE
Question about combustion models

### DISCUSSION THREAD

**Discussion by Pockjob** [AMReX-Combustion/PeleC#785] [Q&A] [answered]
I notice that the chemical source term is solved by a traditional finite-rate chemistry approach. Can turbulence-combustion models such as PaSR  be applied to PeleC?

**Comment by baperry2** [comment, marked as answer]
PeleC takes its chemistry models from PelePhysics, which only supports finite rate chemistry. I do have a branch of PelePhysics that supports flamelet/presumed PDF models for PeleLMeX, but this capability has not been brought to PeleC due to the added complexity of compressibility. There is no support for PaSR models in PelePhysics.
